### ISSUE TITLE
Installation documentation uses undocumented feature

### ISSUE THREAD

**Issue by breyed** [yarnpkg/berry#4507]

**Comment by arcanis** [comment]
It creates and configure the project using the modern versions of Yarn. Without that, for backward compatibility, the created project will use whatever is your global binary, which is likely outdated.

The flag should probably be listed in `yarn init`'s documentation to clear up confusion, indeed.

**Comment by SamB** [comment]
Yes, it should probably be listed at one or both of https://yarnpkg.com/cli/init (because it's used by Yarn 2+ users) and https://classic.yarnpkg.com/en/docs/cli/init (because it's actually implemented as part of Yarn 1.x).
